tests/ui/test_login.py

```diff
@@ -1,11 +1,25 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
+from seleniumlogin import force_login
```

Comment:
Sure, well, I already answered it 😆 it's coming from `django-selenium-login` which I didn't see in the requirements you added :)